traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/edge/CacheRegister.java

```diff
@@ -129,6 +130,12 @@ public void setCacheMap(final Map<String,Cache> map) {
 		return allCaches;
 	}
 
+	public Set<DeliveryServiceMatcher> getDeliveryServiceMatchers(final DeliveryService deliveryService) {
+	    return this.deliveryServiceMatchers.stream()
+				.filter(deliveryServiceMatcher -> deliveryServiceMatcher.getDeliveryService() == deliveryService)
```

Comment:
Do you purposely mean to compare the references here using `==` rather than `getDeliveryService().getId().equals(deliveryService.getId())` to compare the xml_id?

Reply:
Nope! Comparing xml_ids in 1f9d31e444.